src/webnn_native/ops/Squeeze.h

```diff
@@ -35,6 +37,59 @@ namespace webnn_native { namespace op {
             return graph->AddSqueeze(this);
         }
 
+        MaybeError CalculateShape() {
+            auto inputShape = mInputs[0]->Shape();
+            auto inputRank = inputShape.size();
+
+            std::unordered_set<int32_t> axesToSqueeze;
+            if (mAxes.empty()) {
+                for (size_t i = 0; i < inputRank; ++i) {
+                    if (inputShape[i] == 1) {
+                        axesToSqueeze.insert(i);
+                    }
+                }
+            } else {
+                for (const auto& axis : mAxes) {
+                    axesToSqueeze.insert(axis);
+                }
+            }
+
+            std::vector<int32_t> outputShape;
+            for (size_t i = 0; i < inputRank; ++i) {

```

Comment:
The loop is unnecessary for `mAxes.empty()` that can implement directly in L.47.

Reply:
Thanks, I will update. @fujunwei